src/utils/utils.tsx

```diff
@@ -132,3 +132,9 @@ export function kebabUtenSpesialtegn(string: string | {label: string}) {
     const regExpr = /[^a-zA-Z0-9-. ]/g;
     return kebabCase(string).replace(regExpr, '');
 }
+
+export function capitalize(str: string) {
+    return str
+        .toLowerCase()
+        .replace(/(^|[^a-z\u00C0-\u017F\u0400-\u04FF'])([a-z\u00C0-\u017F\u0400-\u04FF])/g, s => s.toUpperCase());
```

Comment:
is this capitalizing all letters or only beginning letters

Reply:
It will fix all letters 👌 
Taken from: https://github.com/navikt/veilarbportefoljeflatefs/commit/106e62a52d5b6e5807282aa6c8d8086f46524ccb

Reply:
Only first letter it seems 😁

Reply:
Ohh so it does not work?

Reply:
I change the function name, maybe it is more clear now?